python/mxnet/ndarray/register.py

```diff
@@ -26,6 +26,7 @@
 from ..base import mx_uint, check_call, _LIB, py_str, _init_op_module, _Null # pylint: disable=unused-import
 
 
+# pylint: disable=too-many-locals
```

Comment:
Obsolete due to global disable

Reply:
It defies explanation but these are actually required or the tests fail.